changelogs/fragments/58-encode-collection-info

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - Encode special characters in collection_info when uploading collection file to artifactory to prevent truncation (https://github.com/briantist/galactory/issues/58)
```

Comment:
```suggestion
  - Encode special characters in ``collection_info`` when uploading collection file to artifactory to prevent truncation (https://github.com/briantist/galactory/issues/58).
```